hbase-server/src/main/java/org/apache/hadoop/hbase/SplitLogCounters.java

```diff
@@ -32,8 +32,6 @@
   public final static LongAdder tot_mgr_log_split_batch_start = new LongAdder();
   public final static LongAdder tot_mgr_log_split_batch_success = new LongAdder();
   public final static LongAdder tot_mgr_log_split_batch_err = new LongAdder();
-  public final static LongAdder tot_mgr_new_unexpected_wals = new LongAdder();
```

Comment:
Remove HBASE-24790  related change in this PR.

Reply:
Thanks for view @pankaj72981  

I will remove the HBASE-24790 related change in this PR